src/ClaimService.Models.Db/DbChangeTemplate.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using LT.DigitalOffice.ClaimService.Models.Dto.Enums;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace LT.DigitalOffice.ClaimService.Models.Db;
+
+public class DbChangeTemplate
+{
+  public const string TableName = "ChangeTemplates";
+
+  public Guid Id { get; set; }
+  public ChangeType ChangeType { get; set; }
+  public Locale Locale { get; set; }
+  public string Content { get; set; }
+
+  public DbClaimChange ClaimChange { get; set; }
```

Comment:
You have relationship but there is no ClaimChangeId which will be used to identify the reference